deploy/admin_clusterrole_binding.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: work-role
+rules:
+  - apiGroups: ["multicluster.x-k8s.io"]
+    resources: ["works", "appliedworks"]
+    verbs: [ "*" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: work-controller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
+subjects:
+  - kind: ServiceAccount
+    name: work-controller-sa
+    namespace: fleet-system
+---
```

Comment:
why do we need the admin role for our sa?

Reply:
Would not the SA need access to those cluster scope resources? Our controllers' dynamic client will need to interact with those types.